requirements-dev.txt

```diff
@@ -5,7 +5,8 @@
 # for now keep pinning "known working" lxml,
 # it's been a troublesome component in the past.
 # Skip  lxml for python 3.11+ on win32 as there's no binary wheel as of 01/22/2023
-lxml==4.9.1 ; python_version < '3.11' and sys_platform != 'win32'
+lxml==4.9.2 ; python_version < '3.12' and sys_platform == 'win32'
+lxml==4.9.2 ; sys_platform != 'win32'
```

Comment:
I'd have made this: on non-windows, block off 3.12 (there won't be wheels until 3.12 releases in October, and then usually several more months to wait.  On windows, don't install it - the tests that need it fail anyway.